onebusaway-android/src/main/java/org/onebusaway/android/ui/PreferencesActivity.java

```diff
@@ -112,6 +114,10 @@ public void onCreate(Bundle savedInstanceState) {
         aboutPref = findPreference(getString(R.string.preferences_key_about));
         aboutPref.setOnPreferenceClickListener(this);
 
+        zipPref = findPreference(getString(R.string.preference_key_zip_code));
```

Comment:
You only need this code if we want to track taps on this preference in analytics via  `onPreferenceClick()` - which I think we do.  Could you add an IF statement and analytics labels for this?  You can use the `aboutPref` as a template.